src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -795,6 +795,9 @@ class ZStackAdapter extends Adapter {
                 };
 
                 this.emit('deviceJoined', payload);
+            }
+            else if (object.command.name === 'srcRtgInd') {
+                this.emit('sourceRoute', object.payload);
```

Comment:
I found this from the debugger. Unfortunately, I don't have any other adapter types at hand to see if they support this.

Reply:
The closest "equivalent" in ember would probably be this:
https://github.com/Koenkk/zigbee-herdsman/blob/855fe7a51fd67ed4fc692d91bba9e3603d326da7/src/adapter/ember/ezsp/ezsp.ts#L5390-L5414